stdlib/dict/dict.flux

```diff
@@ -0,0 +1,19 @@
+package dict
+
+// fromList will convert an array of key/value pairs
+// into a dictionary.
+builtin fromList : (pairs: [{ key: K, value: V }]) => [K: V] where K: Comparable
+
+// get will retrieve a value from a dictionary. If there is no
+// key in the dictionary, the default value will be returned.
+builtin get : (from: [K: V], key: K, default: V) => V where K: Comparable
```

Comment:
I commented on the original design issue but I think being consistent with the name of the `dict` type argument  is important. 

I recommend we rename `from` to `dict` here.

Reply:
I forgot to update both issues, but I'm also fine if we change this to `dict`.